sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/implementation/EncryptionProcessor.java

```diff
@@ -243,6 +243,10 @@ public EncryptionSettings getEncryptionSettings() {
     }
 
     public Mono<byte[]> encrypt(ObjectNode itemJObj) {
+        return encryptObjectNode(itemJObj).flatMap(encryptedObjectNode -> Mono.just(EncryptionUtils.serializeJsonToByteArray(EncryptionUtils.getSimpleObjectMapper(), encryptedObjectNode)));
```

Comment:
you don't need flatMap here. 

Here you are wrapping the result with `Mono.just` then you flatten it with `flatMap` you don't need `flatMap `nor the `Mono.just `here

this is like doing `sqrt(x^2)` just do `x` :-)

```suggestion
        return encryptObjectNode(itemJObj).map(encryptedObjectNode -> EncryptionUtils.serializeJsonToByteArray(EncryptionUtils.getSimpleObjectMapper(), encryptedObjectNode));
```

Reply:
Thanks for noticing, it was copy paste during refactoring from previous code, moved to map now